### ISSUE TITLE
Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond) is not supported

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1468]
While exploring dry-rb stuff, I've met with an unsupported time unit `milisecond` in `Process.clock_gettime` in `dry-monitor` (see https://github.com/dry-rb/dry-monitor/blob/master/lib/dry/monitor/notifications.rb#L13)

According to Ruby documentation is this option is already available since Ruby 2.1 https://ruby-doc.org/core-2.1.1/Process.html#method-c-clock_gettime

To reproduce
```ruby
Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond)
```

Error:

```
RuntimeError: unsupported time unit millisecond
	from (irb):1:in `clock_gettime'
	from (irb):1
	from ~/.rubies/truffleruby-1.0.0-rc9/bin/irb:29:in `<main>'
```

**Comment by deepj** [comment]
It seems `float_millisecond` and  `second` time unit options are not supported as well.

**Comment by nirvdrum** [comment]
Thanks again for reporting  this. It's been fixed as of b19a12d9d10720077e629476aa83d08a878f7d8f and will be part of the next TruffleRuby release.
